### DISCUSSION TITLE
github.copilot.chat.experimental.codeGeneration.instructions, what are the setting options? "enabled"? true? Unclear.

### DISCUSSION THREAD

**Discussion by evilgeniuscreative** [community/community#138305] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am trying to use the new experimental version github.copilot.chat.experimental.codeGeneration.instructions in VS code and copilot, but I can't seem to make it work because I don't know the options for using yet and I don't see them anywhere documented.

What's the proper way to enable this?
  "github.copilot.chat.experimental.codeGeneration.instructions":"array",
  "github.copilot.chat.experimental.codeGeneration.instructions":"enabled",
  "github.copilot.chat.experimental.codeGeneration.instructions":true,

or something else? 

Thanks

**Comment by AlexanderLanin** [comment, marked as answer]
Have a look at https://github.com/microsoft/vscode-docs/blob/main/release-notes/v1_93.md#code-generation-instructions

**Reply by evilgeniuscreative** [reply]
Thank you @AlexanderLanin that worked. But I was unable to make this one to show up as a recognized command (unknown configuration setting) for some reason, despite looking through that document.

`  "github.copilot.config.experimental.inlineChatCompletionTrigger.enabled": true,`

Still "greyed" out. 

Thoughts or suggestions? Seems like some of these new experimental config settings are not yet included in the updates?

Thanks

**Comment by good-dev-student** [comment]
There are any shortcut to toggle enable / disable or i have to develop an extension for that ? thank you

**Comment by good-dev-student** [comment]
OFF:
![image](https://github.com/user-attachments/assets/63d363c3-c121-43f8-9cfd-a6fb65beb284)


ON :
![image](https://github.com/user-attachments/assets/8f35806c-691c-4715-8203-1b4d33d9524c)

**Comment by good-dev-student** [comment]
you can add 2 aliases :

```bash
alias o='cp -f ~/on-settings.json .vscode/settings.json'
alias f='cp -f ~/off-settings.json .vscode/settings.json'
```
